src/main/scala/com.socrata.tileserver/services/TileService.scala

```diff
@@ -169,10 +168,10 @@ object TileService {
                     select: String): Map[String, String] = {
     val params = req.queryParameters
     val whereParam =
-      if (params.contains("$where")) s"""(${params("$where")}) and (${where})""" else where
+      if (params.contains(s"$$where")) s"""(${params(s"$$where")}) and (${where})""" else where
```

Comment:
Hrm.  Grumble grumble grumble, warnings that require changes that make code confusing are just stupid.